{"metadata":{"title":"recoverySuggestion","platforms":[{"name":"iOS","introducedAt":"8.0","deprecated":false,"unavailable":false,"beta":false},{"name":"macOS","introducedAt":"10.10","deprecated":false,"unavailable":false,"beta":false},{"unavailable":false,"name":"tvOS","introducedAt":"9.0","deprecated":false,"beta":false},{"name":"watchOS","deprecated":false,"unavailable":false,"beta":false,"introducedAt":"2.0"}],"modules":[{"relatedModules":["Foundation"],"name":"Auth0"}],"externalID":"s:10Foundation14LocalizedErrorPAAE18recoverySuggestionSSSgvp::SYNTHESIZED::s:5Auth015ManagementErrorV","symbolKind":"property","roleHeading":"Instance Property","role":"symbol","extendedModule":"Foundation","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError\/recoverySuggestion"},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"sections":[],"schemaVersion":{"minor":3,"major":0,"patch":0},"variants":[{"paths":["\/documentation\/auth0\/managementerror\/recoverysuggestion"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.recoverySuggestion"},{"type":"text","text":"."}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError\/LocalizedError-Implementations"]]},"references":{"doc://com.auth0.Auth0/documentation/Auth0/ManagementError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError\/LocalizedError-Implementations","type":"topic","url":"\/documentation\/auth0\/managementerror\/localizederror-implementations","kind":"article"},"doc://com.auth0.Auth0/documentation/Auth0/ManagementError":{"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ManagementError"}],"navigatorTitle":[{"text":"ManagementError","kind":"identifier"}],"abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Management API v2."}],"url":"\/documentation\/auth0\/managementerror","title":"ManagementError","role":"symbol","type":"topic"},"doc://com.auth0.Auth0/documentation/Auth0":{"role":"collection","type":"topic","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","url":"\/documentation\/auth0","title":"Auth0"},"doc://com.auth0.Auth0/documentation/Auth0/ManagementError/recoverySuggestion":{"kind":"symbol","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/ManagementError\/recoverySuggestion","url":"\/documentation\/auth0\/managementerror\/recoverysuggestion","abstract":[],"type":"topic","title":"recoverySuggestion"}}}